specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -4419,6 +4419,14 @@
           "readOnly": true,
           "type": "array"
         },
+        "tactics": {
```

Comment:
Not sure whether this is a good practice to have 2 'tactics' with the same definition.

Reply:
I'm don't completely understand the 2 'tactics' with the same definition.
There's one definition for CaseProperties which is a property of Case that is to be deprecated in a few months, and a definition for IncidentAdditionalData which is a property of Incident which is the one to replace Case.
If it still doesn't make sense can you elaborate?

Reply:
I see. thanks